src/python/pants/backend/codegen/thrift/python/apache_thrift_py_gen.py

```diff
@@ -0,0 +1,28 @@
+# coding=utf-8
+# Copyright 2016 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import os
+
+from pants.backend.codegen.thrift.lib.apache_thrift_gen_base import ApacheThriftGenBase
+from pants.backend.codegen.thrift.python.python_thrift_library import PythonThriftLibrary
+from pants.backend.python.targets.python_library import PythonLibrary
+
+
+class ApacheThriftPyGen(ApacheThriftGenBase):
```

Comment:
True, although that's honored more in the breach than in the observance, if my spot-checking of other tasks is anything to go by... Will add something here, but we'll need to do a sweep through all the tasks.